cmd/root_cmd.go

```diff
@@ -14,56 +18,84 @@ var (
 		
 Full documentation available at https://docs.koki.io/short
 `,
-		RunE: short,
+		RunE:         short,
+		SilenceUsage: true,
 		Example: `
-#Find the shorthand representation of kubernetes objects
-short man pod							#opens man page for pod syntax and conversion
-short man deployment						#opens man page for deployment syntax and conversion
+  # Find the shorthand representation of kubernetes objects
+  short man pod
+  short man deployment
 
-#Convert existing kubernetes manifestes to shorthand format
-short -f pod.yaml						#converts the contents of pod.yaml to shorthand syntax
+  # Convert existing kubernetes manifestes to shorthand format
+  short -f pod.yaml
 
-#Stream in manifest files
-cat pod.yaml | short -						#reads from stdin instead of from an input file
+  # Stream in manifest files
+  cat pod.yaml | short -
 
-#Read from url
-short -f http://spec.com/pod.yaml				#reads from URL instead of from an input file
+  # Read from url
+  short -f http://spec.com/pod.yaml
 
-#Convert shorthand file to native syntax
-short --kube-native -f pod_short.yaml				#converts the contents of pod_short.yaml to native syntax
-short -k -f pod_short.yaml					#converts the contents of pod_short.yaml to native syntax
+  # Convert shorthand file to native syntax
+  short --kube-native -f pod_short.yaml
+  short -k -f pod_short.yaml
 
-#Output to file 
-short -f pod.yaml -o pod_short.yaml				#prints the output into file named pod_short.yaml
+  # Output to file 
+  short -f pod.yaml -o pod_short.yaml	
 `,
 	}
 
-	//verbosity denotes the amount of information output as log info messages
-	verbosity int
-	//kubeNative denotes that the conversion must output in kubernetes native syntax
+	// kubeNative denotes that the conversion must output in kubernetes native syntax
 	kubeNative bool
-	//filenames holds the input files that are to be converted to shorthand or kuberenetes native syntax
+	// filenames holds the input files that are to be converted to shorthand or kuberenetes native syntax
 	filenames []string
-	//output denotes the destination of the convereted data
+	// output denotes the destination of the convereted data
```

Comment:
converted

Reply:
Fixed it in the 3rd PR. (wire in output). Otherwise I'll have to rebase a bunch of times, and that is time consuming.